### ISSUE TITLE
Refreshing list of timers adding or removing problem 

### ISSUE THREAD

**Issue by luidiva** [siddii/angular-timer#316]
I have a **ons-list** with **ng-repeat** and a refresh button that reloads all list items bringing more from server.
When time reaches zero or when I add a new timer at the top of the list, all timers get confused (the new one get the second item value and so). Only works when I reload the Controller.
I tried using 1.2 and last versions and the problem happens in both.

`<ons-list ng-repeat="(item, itemActiveTimer) in activeTimers">
<ons-list-item>
<timer  interval="1000" max-time-unit="'hour'" countdown="itemActiveTimer.TimeLeft" finish-callback="callbackTimer.finished(item, itemActiveTimer)">{{hhours}}:{{mminutes}}:{{sseconds}}</timer>
`

**Comment by luidiva** [comment]
The problem was I was using $rootScope to load activeTimers items. I had to manage to have everything inside $scope.

**Comment by luidiva** [comment]
I've found a solution, but every time I call "timer-reset" I receive the message in console:

_ReferenceError {stack: "ReferenceError: timeoutId is not defined↵    at m.…www/vendors/onsen/js/angular/angular.min.js:51:68", message: "timeoutId is not defined"}_

The solution is call this after I update timer items:

`$timeout(function(){
                    $scope.$broadcast('timer-reset');
                    $scope.$broadcast('timer-start');
                });`
Can anyone approve this solution, please?

**Comment by vinoadth** [comment]
No, I still get the same error when I use your code i.e. $timeout(function(){ $scope.$broadcast('timer-reset'); $scope.$broadcast('timer-start'); });
